gensim/test/test_dl4j_wrapper.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Mohit Rathore <mrmohitrathoremr@gmail.com>
+# Copyright (C) 2010 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+
+"""
+Automated tests for checking transformation algorithms (the models package).
+"""
+
+
+import logging
+import unittest
+import os
+import tempfile
+import numpy
+from gensim.models.wrappers.dl4j import dl4jwrapper
+
+module_path = os.path.dirname(__file__) # needed because sample data files are located in the same folder
+datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
+
+def testfile():
+    # temporary model will be stored to this file
+    return os.path.join(tempfile.gettempdir(), 'gensim_dl4j.test')
+
+class TestDl4j(unittest.TestCase):
+    def setUp(self):
+        self.corpus_file = datapath('raw_sentences.txt')
+        self.out_path = 'testmodel'
+        self.test_model_file = datapath('raw_dl4j.txt')
```

Comment:
Since it is in glove format it will be similar to https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/test/test_wordrank_wrapper.py#L34